pennylane/pauli/pauli_arithmetic.py

```diff
@@ -377,10 +378,11 @@ def hamiltonian(self, wire_order=None):
         if len(self) == 0:
             if wire_order in (None, [], wires.Wires([])):
                 raise ValueError("Can't get the Hamiltonian for an empty PauliSentence.")
-            return Hamiltonian([1], [Identity(wires=wire_order)])
+            return Hamiltonian([0], [Identity(wires=wire_order)])
```

Comment:
Technically `qml.Hamiltonian([], [])` would also be a valid return here.

Reply:
Ahh that's elegant! Do we have a preference? I feel like the 0 case is a bit more verbose. This however could be a nice standard to have as well.

Reply:
Probably doesn't matter too much since it will go away in the long term. Can you think of any cases where an empty hamiltonian would cause problems?

Reply:
Not really, It is the current default. I will swap over to that 👍🏼